test/configuration_test.go

```diff
@@ -0,0 +1,42 @@
+package test
+
+import (
+	"runtime"
+
+	"github.com/confluentinc/cli/internal/pkg/config"
+)
+
+func (s *CLITestSuite) TestConfigurationList() {
+	cfg := config.AuthenticatedConfigMockWithContextName("context1")
+	if err := cfg.Save(); err != nil {
+		panic(err)
+	}
+
+	test := CLITest{args: "configuration list", fixture: "configuration/list.golden", workflow: true}
+	if runtime.GOOS == "windows" {
+		test.fixture = "configuration/list-windows.golden"
+	}
+	s.runIntegrationTest(test)
+}
+
+func (s *CLITestSuite) TestConfigurationSet() {
+	cfg := config.AuthenticatedConfigMockWithContextName("context1")
+	if err := cfg.CreateContext("context2", "http://test", "costa", "rica"); err != nil {
+		panic(err)
+	}
+	if err := cfg.Save(); err != nil {
+		panic(err)
+	}
+
+	tests := []CLITest{
+		{args: "configuration set disable_update_check=true", fixture: "configuration/set-one.golden"},
+		{args: "configuration set disable_update_check=false disable_plugins=true current_context=context2", fixture: "configuration/set-multiple.golden"},
```

Comment:
Do you think editing multiple configurations is an actual use case?

Reply:
I mainly decided this since it follows from the style of the `cluster config update` and `topic update`. I do think since `update` commands across the CLI allow for the user to update multiple things at a time, this command should too for consistency (especially if it's called `update`).

If we only want to set 1 field at time, then we probably should call the command `set` and instead take in 2 args: `key` and `value`.

Reply:
> it follows from the style of the cluster config update and topic update

Those commands use a `--config` flag though, which you can pass a comma-delimited list to

Reply:
> If we only want to set 1 field at time, then we probably should call the command set and instead take in 2 args: key and value.

This sounds a bit better and matches https://jamesdefabia.github.io/docs/user-guide/kubectl/kubectl_config_set/